lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -317,6 +314,10 @@ def process_hck_network
       @nm_commands_start << bridge_start
       @nm_commands_stop << bridge_stop
 
+      bridge_start, bridge_stop = @nm.test_bridge_command
```

Comment:
After these changes, we always will create a test bridge even not use it. Not sure if this is good or not.

Reply:
That behavior and the reasoning behind is explained in the commit message. I hope it makes sense, but we can always make an additional change if it turns out otherwise.

> After this change, the network test bridge will be always created even if network tests are not performed, but the overhead should be negligible, considering the cost of the control bridge is ~torelated~ tolerated.